index.html

```diff
@@ -64,10 +64,10 @@ <h3>
                 First Week
               </h3>
                 <ul class="languages-grid">
-                        <li class="lang-list l1"><strong>Html5</strong></li>
-                        <li class="lang-list l2"><strong>Css3</strong></li>
-                        <li class="lang-list l3"><strong>Git</strong></li>
-                        <li class="lang-list l4"><strong>Github</strong></li>
```

Comment:
Fixed _accessibility issue_ 1.
 - These lines, as well as the following that have the same listed items are "styled" with the `strong` tag. And not only was a wrong use of the tag but also it was bad for accessibility. So that's why they were removed.